### DISCUSSION TITLE
The beta news feed is just too much

### DISCUSSION THREAD

**Discussion by globules-io** [community/community#13147] [Other Feature Feedback, Questions, & Ideas]
I do not need to see recommendations, nor activity of people I don't follow. It looks like github is trying to push "social" features but we're here to work and get to the point. The current feed is all I need. The beta feed offers nothing of value. Don't fix what's not broken.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
